services/src/main/java/org/apache/unomi/services/services/ProfileServiceImpl.java

```diff
@@ -862,6 +923,7 @@ private void loadPredefinedPropertyTypes(BundleContext bundleContext) {
             return;
         }
 
+        List<PropertyType> allPropertyTypes = new ArrayList<>();
```

Comment:
This variable name is confusing I think it would be better to call it bundlePropertyTypes

Reply:
You are right, I will do that.